source/includes/stackpath/_network_policy_rules.md

```diff
@@ -52,17 +52,22 @@ curl -X GET \
 
 Retrieve a list of all network policy rules in a given [environment](#administration-environments).
 
+
+Query Params | &nbsp;
+---- | -----------
+`workloadId`<br/>*string* | The workload ID to get the network policy rules for. It is optional.
+
```

Comment:
I missed the query params in my docs PR, thanks for catching that. We can also filter by `type`.

```suggestion
 `type`<br/>*string* | The type of network policy rule, either `INBOUND` or `OUTBOUND`. It is optional.
```